recipes/libnettle/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "nettle" %}
+{% set version = "3.3" %}
+{% set sha256 = "46942627d5d0ca11720fec18d81fc38f7ef837ea4197c1f630e71ce0d470b11e" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://ftp.gnu.org/gnu/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - configure.patch
+
+build:
+  number: 0
+  skip: True  # [not linux]  
+
+requirements:
+  build:
+    - m4
+    - gcc
+  run:
+    - gmp
```

Comment:
Should this be a build time dependency as well? It probably should be [pinned]( https://github.com/conda-forge/conda-forge.github.io/blob/fbd5063c2ddac6f81fef061557de27f71f0827c6/scripts/pin_the_slow_way.py#L50 ) as well.